src/handlers/webhooks/get_webhooks.rs

```diff
@@ -15,25 +15,31 @@ use {
 pub async fn handler(
     Path(project_id): Path<String>,
     State(state): State<Arc<AppState>>,
-) -> std::result::Result<axum::response::Response, crate::error::Error> {
+) -> Result<impl IntoResponse> {
     info!("Getting webhooks for project: {}", project_id);
 
-    let mut webhooks = HashMap::new();
-
-    let mut cursor = state
+    let cursor = state
         .database
         .collection::<WebhookInfo>("webhooks")
         .find(doc! {"project_id": project_id}, None)
         .await?;
 
-    // Iterate over cursor adding webhook id:webhookconfig pairs to hashmap
-    while let Some(webhook) = cursor.try_next().await? {
-        let webhook_config = WebhookConfig {
-            url: webhook.url,
-            events: webhook.events,
-        };
-        webhooks.insert(webhook.id, webhook_config);
-    }
+    let webhooks: HashMap<_, _> =
+        cursor
+            .try_collect()
```

Comment:
You can `.map()` each element of the stream, and then call `try_collect`. This way it won't be necessary to re-allocate the vector.